project_task_description_template/models/project_task.py

```diff
@@ -0,0 +1,38 @@
+# Copyright (C) 2023 Cetmix OÜ
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/lgpl)
+
+from odoo import api, fields, models
+
+
+class ProjectTask(models.Model):
+    _inherit = "project.task"
+
+    task_template_id = fields.Many2one(
+        string="Template", comodel_name="project.task.template"
+    )
+
+    template_visible = fields.Boolean(compute="_compute_template_visible")
+
+    @api.depends("project_id")
```

Comment:
This function should also depend on `stage_id`

Reply:
Thanks for the feedback, I'll take your comments into account and fix everything.